docs/content/en/docs/pipeline-stages/testers/custom.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Custom Test"
+linkTitle: "Custom Test"
+weight: 20
+featureId: test.custom
+---
+
+
+Custom Test would allow developers to run custom commands as part of the development pipeline. The command will be executed in the testing phase of the Skaffold pipeline. It will run on the local machine where Skaffold is being executed and works with all supported Skaffold platforms. Users can opt out of running custom tests by using the `-skipTests` flag.
```

Comment:
Can you state somewhere that `Custom Test` run after each image rebuild. It's not immediately apparent when they run.